src/components/helmetComponent/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import Helmet from 'react-helmet'
+import PropTypes from 'prop-types'
+
+const HelmetComponent = ({ title, description }) => (
+  <Helmet
+    title={`${title}| Pride in London`}
```

Comment:
See other comment about the missing space. Where possible, practice Test Driven Development (TDD) to avoid issues like this.